### PR TITLE
fix(sdk): aws funtion invoke returns the correct type

### LINKED ISSUE #2559: Function invoke test works on sim but not on tf-aws
### I tried this:

Run the invoke function test with tf-aws target breaks, but works on sim

### This happened:

image

### I expected this:

Works

### Is there a workaround?

I don't know

### Component

Compiler

### Wing Version

0.17.7

### Wing Console Version

latest

### Node.js Version

v18.12.1

### Platform(s)

Linux

### Anything else?

_No response_

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

### PR DESCRIPTION
**Pull request by marciocadev**
Closes #2559 

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: libs/wingsdk/src/shared-aws/function.inflight.ts [hunk 1]
```diff
@@ -27,6 +27,6 @@ export class FunctionClient implements IFunctionClient {
     if (!response.Payload) {
       return "";
     }
-    return toUtf8(response.Payload);
+    return JSON.parse(toUtf8(response.Payload));
```
Comment (Chriscbr) [inline libs/wingsdk/src/shared-aws/function.inflight.ts:30]: If we just add this line (without changing the signature of `invoke` in `cloud/function.ts`) does the fix still work?

Comment (marciocadev) [reply libs/wingsdk/src/shared-aws/function.inflight.ts:30]: To be true, I made this change to give the possibility to return any type, not only string

As the documentation of cloud.Function it's declare as any

Comment (marciocadev) [reply libs/wingsdk/src/shared-aws/function.inflight.ts:30]: To be honest, I made this change after reviewing the documentation (where the invoke function returns a JSON), and this modification allows for the return of any type of value.

**Review by Chriscbr — APPROVED** [review]
🙏

**Pull request merged**